publish/defs.bzl

```diff
@@ -125,17 +125,12 @@ def _artifact_bundle_impl(ctx):
     # List of input files
     input_files = ctx.files.inputs
 
-    bundle_dir = "{}_bundle".format(ctx.attr.name)
+    bundle_root = ctx.actions.declare_directory("bundle-{}".format(ctx.attr.name))
```

Comment:
would be ideal if we could instead append a hash of the name here (we otherwise end up with `bundle-bundle` if `name` is `bundle`) but starlark doesn't have a hash primitive